app/models/audit/eligibility/EligibilityCheck.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.audit.eligibility
+
+import essttp.rootmodel.ttp.ChargeTypeAssessment
+import play.api.libs.json.{Json, OWrites}
+
+final case class EligibilityCheck(
```

Comment:
I know it's in the audit directory which I think is good enough, but maybe renaming this to EligibilityCheckAuditDetail or something - just to be crystal clear. I'm not overly fussed though, so feel free to completely ignore 😄

Reply:
👍 clearer is usually better - updated [here](https://github.com/hmrc/essttp-frontend/pull/92/commits/2494869cf8e983d94983c15f96ec8a49d8ff40a2)